Presentation.Web/Tests/PageObjects/it-contract/ItContractOverview.po.ts

```diff
@@ -13,5 +13,7 @@ class ItSystemEditPo implements PageObject {
     navigationBarWrapper = new navigationBarWrapper();
     subNavigationBarWrapper = new subNavigationBarWrapper();
 
+    contractName = element(by.id("name"));
+    saveContractBtn = element(by.buttonText("Gem"));
```

Comment:
Findes der en bedre løsning end by.buttonText?

Reply:
Deleted not used